src/screens/Components.js

```diff
@@ -284,6 +284,57 @@ export default class Components extends React.Component {
                 >
                   <LinearGradient colors={['transparent', 'rgba(0,0,0, 0.8)']} style={styles.cardGradient} />
                 </Card>
+                <Block width={330}>
```

Comment:
Every component has its own `<Block />` tag for better semantics, please use the same way of grouping the components as used before. Sliders must have their own `<Block />`, Checkboxes need that as well. ( not every component per say, more like every group of the same components -- So, 3 checkboxes have their own `<Block />` component, not 3x `<Block />` ).

Reply:
Instead of using fixed numbers -- because of the different screen sizes -- it's better to use Block as such: `<Block flex style={{ width: '100%' }} />`